source/designers-guide/preparing-themes-for-the-community-store/index.md

```diff
@@ -25,24 +25,87 @@ The plugin directory has to have a specific structure in order to work inside Sh
 [developer prefix][plugin name]
 ```
 
-Themes in the Shopware Community Store have to be wrapped inside plugins in order to be installable with the plugin manager. The plugin requires a `Bootstrap.php` file in the root directory and the custom theme, that was previously created. It has to be located inside the `Themes/Frontend` directory (just as it would be inside the normal Shopware installation).
+## 5.2 plugin system
+
+Themes in the Shopware Community Store have to be wrapped inside plugins in order to be installable with the plugin manager. The plugin requires a *plugin base file* and a `plugin.xml` file in the root directory and the custom theme, that was previously created. It has to be located inside the `Resources/Themes/Frontend` directory (just as it would be inside the normal Shopware installation).
 
 **Attention: the directory and file names are case sensitive.**
 
+##### Plugin directory
+```
+SwagTutorialTheme
+ ├── Resources
+ │    ├── Themes
+ │    │    ├── Frontend
+ │    │    │    ├──TutorialTheme
+ │    │    │    │    ├── preview.png
+ │    │    │    │    ├── Theme.php
+ │    │    │    │    └── frontend
+ ├── SwagTutorialTheme.php
+ └── plugin.xml
+```
+
+### Creating the plugin
+
+Just create a plain *plugin base file*. Because the `TutorialTheme` directory is located inside the `Resources/Themes/Frontend`, the plugin automatically detects its content.
+
+##### SwagTutorialTheme.php - *the plugin base file*
+```php
+<?php
+namespace SwagTutorialTheme;
+
+use Shopware\Components\Plugin;
+
+class SwagTutorialTheme extends Plugin
+{
+}
+```
+
+You need to create a `plugin.xml` file in the same directory, which defines the meta data of your plugin, i.e. label, version, compatibility or the changelog.
+
+##### plugin.xml
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<plugin xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/shopware/5.5/engine/Shopware/Components/Plugin/schema/plugin.xsd">
```

Comment:
if the minVersion of the plugin is SW 5.2.0 we usually use the same xsd path here. so the path to the 5.2 version